### DISCUSSION TITLE
[APPROVED] GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by markwillcraft** [community/community#76538] [GitHub Education] [answered]
### Good day, everyone!,

My application has been approved, but I'm still unable to access the benefits. Can someone assist me?
<img width="366" alt="image" src="https://github.com/community/community/assets/78254466/3f2924be-231f-4bf5-a61d-7329f84051eb">

**Comment by Yee-Taung** [comment]
You will receive an email with title `[GitHub Education] 💖 @username` few days after the approval. The benefits can be accessed once you receive the email. Hope it helps.

**Reply by markwillcraft** [reply]
Is it guaranteed that I will receive it? I've come across posts where individuals have been waiting for weeks without receiving the benefits. 🥲

**Comment by samus-aran** [comment, marked as answer]
Hi @markwillcraft 👋 

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!

If you don't receive an email within 5 days since your approved status I would then suggest you re-open this discussion and we would be happy to help further 👍

**Reply by markwillcraft** [reply]
hi @samus-aran, Is it guaranteed that I will receive it? I've come across posts where individuals have been waiting for weeks without receiving the benefits. 🥲

**Reply by markwillcraft** [reply]
Woah! Got it 🤩 Thanks guys

<img width="593" alt="image" src="https://github.com/community/community/assets/78254466/33e11122-bcc3-4d6e-9747-2a4bc30e7465">
<img width="346" alt="image" src="https://github.com/community/community/assets/78254466/47aea0be-dc60-440c-aa9e-d3b8f10ce4aa">
